Documentation/Keyboards/K-Type.md

```diff
@@ -16,7 +16,21 @@
  1) Flip your K-Type over.
  2) With the keys down you should see a small hole on the left side in the middle of the board.
  3) While the keyboard is plugged in, use a paperclip or other small object to press the button inside the case.
+    - TIP: you can also configure a key combination to enter flash mode. On the default layout it's Fn-ESC.
  4) If the LEDs turn off and you see the amber light in the diffuser on the lower right (when looking at the top) it is now in flash mode.
+    - Note: on some keyboards there is no amber light indication (there are no leds at all in flash mode)
+
+#### Failsafe mode
```

Comment:
Not 100% accurate, but still a good way to put it :D 

Basically, if the keyboard fails to flash, it will go into "Secure DFU Mode". Currently there is no way to negotiate a key, so you have one of two options. Once I have HID-IO in a good state there will be a way to negotiate secure keys while flashing the keyboard.
1) Unplug and Replug the keyboard (this will work if the flash failed immediately without writing any data).
2) Press the flash button (should only need to do it once, but it doesn't hurt pressing it more times).

Reply:
Thanks for the information, I'll incorporate it, but I have a question regarding (2). I had a bad flash where the keyboard behaved like this:
- After the flash it remained in flash mode (and also after unplugging and replugging), however trying to flash it would fail *after* dfu-util reached 100%. I tried this several times always with the same result.
- I saw the bit about pressing the flash button twice in the discord help thread, tried it and the next flash immediately succeed.

So it seems to work?